CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 - Introduced support for variable arguments to UX functions - inherit from the `Fuse.Reactive.VarArgFunction` class.
 - The classes `Vector2`, `Vector3` and `Vector4` in `Fuse.Reactive` are now removed and replaced with the general purpose, variable-argument version `Vector` instead. This ensures vectors of any length are treated the same way. This is backwards incompatible in the unlikely case of having used these classes explicitly from Uno code.
 - Added support for name-value pair syntax: `name: value`. Can be used for JSON-like object notation and named arguments in custom functions.
+- Any vector of name-value pairs is now interpreted as an `IObject`, e.g. `{name: 'Joe', apples: 10}`
```

Comment:
Since name-value pairs is new in this release, I think this line is just confusing.